README.md

```diff
@@ -79,6 +80,14 @@ Use `toUpperCase()` to capitalize first letter, `slice(1)` to get the rest of th
 var capitalize = str => str[0].toUpperCase() + str.slice(1);
 ```
 
+### Capitalize first letter of every word
+
+Use `replace()` to match the first character of each word and `toUpperCase()` to capitalize it.
```

Comment:
By the way, this is a very smart technique. Thanks for teaching me something today!